Rules/UseProcessBlockForPipelineCommand.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;

```

Comment:
Need copyright header

Reply:
fixed in #707f02d96ac5763afaf0b69649d787f8f25b2492